addons/stock_picking_wave/wizard/picking_to_wave.py

```diff
@@ -1,16 +1,14 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from openerp.osv import fields, osv
+from openerp import fields, models
 from openerp.tools.translate import _
 
-class stock_picking_to_wave(osv.osv_memory):
+class StockPickingToWave(models.TransientModel):
     _name = 'stock.picking.to.wave'
     _description = 'Add pickings to a picking wave'
-    _columns = {
-        'wave_id': fields.many2one('stock.picking.wave', 'Picking Wave', required=True),
-    }
 
-    def attach_pickings(self, cr, uid, ids, context=None):
+    wave_id = fields.Many2one('stock.picking.wave', 'Picking Wave', required=True)
+
+    def attach_pickings(self):
         #use active_ids to add picking line to the selected wave
-        wave_id = self.browse(cr, uid, ids, context=context)[0].wave_id.id
-        picking_ids = context.get('active_ids', False)
-        return self.pool.get('stock.picking').write(cr, uid, picking_ids, {'wave_id': wave_id})
\ No newline at end of file
+        wave_id = self.wave_id.id
+        return self.env['stock.picking'].write({'wave_id': wave_id})
```

Comment:
decorate this method with @api.multi and change it's code to,

self.ensure_one()
picking_ids = self.env.context.get('active_ids')
return self.env['stock.picking'].browse(picking_ids).write({'wave_id': self.wave_id.id})